go.mod

```diff
@@ -22,11 +21,9 @@ require (
 	github.com/osrg/gobgp v2.0.0+incompatible
 	github.com/prometheus/client_golang v1.0.0
 	github.com/satori/go.uuid v1.2.0 // indirect
-	github.com/spf13/cobra v0.0.5
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/viper v1.3.2 // indirect
```

Comment:
Just curious, what did you run exactly to update this?

Reply:
`go mod tidy`

Reply:
Oh, perfect. Thanks! :)